docker/Dockerfile.flux

```diff
@@ -37,6 +37,11 @@ ENTRYPOINT [ "/sbin/tini", "--", "fluxd" ]
 COPY --from=quay.io/squaremo/kubeyaml:0.7.0 /usr/lib/kubeyaml /usr/lib/kubeyaml/
 ENV PATH=/bin:/usr/bin:/usr/local/bin:/usr/lib/kubeyaml
 
+# Add the sops binary and allow it to be executed
+ARG SOPS_VERSION=3.4.0
+ADD https://github.com/mozilla/sops/releases/download/$SOPS_VERSION/sops-$SOPS_VERSION.linux /usr/local/bin/sops
```

Comment:
Can you install sops verifying its checksum and using the build cache system like, e.g. `kubectl`? 
https://github.com/fluxcd/flux/blob/518933bf8fc906e90e67c5a78439aa6039c1b6be/Makefile#L12
https://github.com/fluxcd/flux/blob/518933bf8fc906e90e67c5a78439aa6039c1b6be/Makefile#L94-L95
https://github.com/fluxcd/flux/blob/518933bf8fc906e90e67c5a78439aa6039c1b6be/Makefile#L109-L114

Reply:
It should be done correctly now, but please check the work. I can't say I fully understand this makefile, so I might have missed something.

Reply:
It is done correctly. Thanks!!